CHANGES

```diff
@@ -1,3 +1,55 @@
+Changes from 8.6 to 8.6.1
+=========================
+
+- Fix #5380: Default colors for CoqIDE are actually applied.
+- Fix plugin warnings
+- Use more neutral wording instead of mentioning CoqIDE
```

Comment:
I'll remove this change from the log if you don't mind, I don't think it brings much to the user to learn about it.